db/migrate/20110601153519_really_regenerate_runtime_dependencies_list.rb

```diff
@@ -19,7 +19,7 @@ def self.each_dependency
       inner join rubygems on rubygems.id = dependencies.rubygem_id
       where scope = 'development'
     SQL
-    connection.select_all(dependencies).each_with_index do |row, index|
+    connection.select_all(dependencies).each_with_index do |row, _|
```

Comment:
It’s an old migration, so it’s code that will never run again anyway. I guess we just keep it around for documentation purposes.